### ISSUE TITLE
Text is blurry

### ISSUE THREAD

**Issue by yunyu** [ghosh/microtip#6]
![](https://i.imgur.com/iRLkIwB.png)

**Comment by ghosh** [comment]
@yunyu Is it blurry for you on the Microtip website too? - https://github.com/ghosh/microtip

**Comment by yunyu** [comment]
@ghosh Nope, but it seems to happen with some other fonts (which make the height/width of the :after an odd number).

**Comment by ghosh** [comment]
@yunyu Ok, it's a quite a common issue caused by CSS transitions. Will look into it.

**Comment by yunyu** [comment]
I think a workaround would be to add some classes that make the paddings in half pixels

**Comment by fireswork** [comment]
+1, i have this problem too

**Comment by Muhnad** [comment]
Hey @yunyu @fireswork

Thanks for submitting the issue but it's will be great if you provide an example of the issue and we will try to fix this ASAP.

**Comment by bilalmalkoc** [comment]
Any update for this problem? It happens with position: absolute and translate3d

**Comment by Muhnad** [comment]
Hey @bilalmalkoc, could you pass me a link, please? it'll help me alot

**Comment by bilalmalkoc** [comment]
@Muhnad hello. Check share buttons on [this ](https://ibdev.wpuzman.xyz/1/30-reasons-you-should-never-let-your-dog-play-in-the-mud/)page please. Change position to right from source coude. It happens only right.

**Comment by ugurcatak** [comment]
translate3d makes it blurry.
`[role~="tooltip"][data-microtip-position="top"]:hover::after {
     transform: translate3d(-50%, -5px, 0);
}
[role~="tooltip"][data-microtip-position|="top"]::after {
    transform: translate3d(-50%, 0, 0);
}`

**Comment by aleksandras-va** [comment]
Omitting `will-change: transform` mitigates the issue. This may impact performance in some way though.
